mne_bids/write.py

```diff
@@ -144,6 +144,127 @@ def _channels_tsv(raw, fname, overwrite=False, verbose=True):
     return fname
 
 
+def _electrodes_tsv(raw, fname, kind, overwrite=False, verbose=True):
+    """
+    Create an electrodes.tsv file and save it.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The data as MNE-Python Raw object.
+    fname : str
+        Filename to save the electrodes.tsv to.
+    kind : str
+        Acquisition type to save electrodes in. For iEEG, requires size.
+    overwrite : bool
+        Defaults to False.
+        Whether to overwrite the existing data in the file.
+        If there is already data for the given `fname` and overwrite is False,
+        an error will be raised.
+    verbose : bool
+        Set verbose output to true or false.
+    """
+    x, y, z, names = list(), list(), list(), list()
+    for ch in raw.info['chs']:
+        if _check_ch_locs([ch]):
+            x.append(ch['loc'][0])
+            y.append(ch['loc'][1])
+            z.append(ch['loc'][2])
+        else:
+            x.append('n/a')
+            y.append('n/a')
+            z.append('n/a')
+        names.append(ch['ch_name'])
+
+    if kind == "ieeg":
+        sizes = ['n/a'] * len(names)
+        data = OrderedDict([('name', names),
+                            ('x', x),
+                            ('y', y),
+                            ('z', z),
+                            ('size', sizes),
+                            ])
+    else:
+        data = OrderedDict([('name', names),
+                            ('x', x),
+                            ('y', y),
+                            ('z', z),
+                            ])
+    _write_tsv(fname, data, overwrite=overwrite, verbose=verbose)
+    return fname
+
+
+def _coordsystem_json(raw, unit, orient, coordsystem_name, fname,
```

Comment:
don't move the function so that the diff is easier to read. Smaller diff is always better :-)